### ISSUE TITLE
Why is the scaleZ attribute still invalid? Is the new code in the npm module? How do I check?

### ISSUE THREAD

**Issue by drubio1989** [facebook/react-360#638]
I'm creating a new project with react 360 and found that the scaleZ attribute is still broken in the version that I have. Referencing the issue that I posted in September ([https://github.com/facebook/react-360/issues/578](url)), it was commented that the new code is not in the npm module yet. How do I check if the code is in the npm module and how do I get it? I'm not familiar with react360, vr, or with node/npm. Some guidance would be great. Thank you.

**Comment by drubio1989** [comment]
Am I correct to assume that it's still not in the npm module yet? I checked on npmjs.org and it says that the version is in 1.0.0 which is one minor verson behind. Can I get confirmation on this? I would like to manipulate my 3D objects but without the scaleZ attribute, it is proving to be difficult. If the code with scaleZ attribute exists for download I would like to be able to get it.

**Comment by kNewbown** [comment]
Nope, it's not in the npm module yet. You can manually update it though, so that you can use it now. I don't know if there is a better way, but this is how I worked around it.

Copy all of the latest code from github for the file [react-360/Libraries/VRReactOverrides/processTransform.vr.js](https://github.com/facebook/react-360/blob/c3e7e7f75abe39bcb90623e8c74963afa483149f/Libraries/VRReactOverrides/processTransform.vr.js)

Delete the contents of the processTransform.vr.js file in your project, and paste in the updated code from github.

Un-ignore the processTransform.vr.js file in the gitignore file:
```
node_modules/react-360/*
!node_modules/react-360/Libraries/
node_modules/react-360/Libraries/*
!node_modules/react-360/Libraries/VRReactOverrides/
node_modules/react-360/Libraries/VRReactOverrides/*
!node_modules/react-360/Libraries/VRReactOverrides/processTransform.vr.js
```

**Comment by fcor** [comment]
Not in the npm module yet, but @kNewbown solution works fine, thanks!
